server/src/actors/api/mod.rs

```diff
@@ -3,11 +3,12 @@ use actix::Addr;
 use super::server::Server;
 
 mod inbound_message;
+pub mod message;
 mod websocket;
 
-pub mod outbound_message;
-pub use crate::api::websocket::WebSocket as ApiClient;
+use websocket::WebSocket;
+pub use websocket::WebSocket as ApiClient;
 
 pub fn create_ws_api(server: Addr<Server>) -> ApiClient {
-    return websocket::WebSocket::new(server);
+    WebSocket::new(server)
```

Comment:
:+1: ...btw, I with Rust had a single expression syntax for function definition like Scala does.. 
```scala
def f(a: A): B = a.intoB
```